docker-compose.yml

```
@@ -712,7 +712,7 @@ services:
 
   # OpenSearch store - node1
   opensearch-node1: # This is also the hostname of the container within the Docker network (i.e. https://opensearch-node1/)
-    image: opensearchproject/opensearch:2.7.0 # Specifying the latest available image - modify if you want a specific version
+    image: opensearchproject/opensearch:2.8.0 # Specifying the latest available image - modify if you want a specific version
```

```suggestion
    image: "opensearchproject/opensearch:${VERSION}" # Specifying the latest available image - modify if you want a specific version
```
Why not use the environment variable here?

---

👍